app/homepage/ui.test.js

```diff
@@ -65,3 +65,54 @@ test('should navigate to the browse solution page when clicking on the browse pr
     .click(browsePromoLink)
     .expect(getLocation()).contains('/solutions');
 });
+
+fixture('Footer')
```

Comment:
While we have done this. Can you please add one for the Header too. Might as well get that ui.tested. You do seem to enjoy writing these 😈